includes/classes/admin/mapping-wizard.php

```diff
@@ -520,6 +520,38 @@ public function map_template() {
 		$sync_items = $mapping_id && $this->_get_val( 'sync-items' );
 		$notes      = '';
 
+		$is_acf_pro_installed = class_exists('acf_pro');
+		$template_has_repeatable_fields = false;
+		if (isset($template->related->structure->groups)) {
+			foreach ($template->related->structure->groups as $group) {
+				if (isset($group->fields)) {
+					foreach ($group->fields as $field) {
+						if (isset($field->metadata->repeatable) && $field->metadata->repeatable->isRepeatable) {
+							$template_has_repeatable_fields = true;
+							break 2;
+						}
+					}
+				}
+			}
+		}
+
+		if ($template_has_repeatable_fields && !$is_acf_pro_installed ){
+			$notes .= $this->view(
+				'graceful-degradation',
```

Comment:
When testing this out without the acf pro I thought this message was meant to appear. However after having repeatable fields I can't see this message appear. Feels like we should definitely be saying something.

<img width="2048" alt="Screenshot 2024-04-15 at 12 32 29" src="https://github.com/fjorgedigital/gathercontent-wordpress-plugin/assets/1649191/0efa3bc9-036b-4ea7-a23f-9099eb252ecf">

Reply:
@CWDN  @HughbertD 

I followed up with access for Chris to our staging site, so we didn't make any updates for this comment as your feedback on Basecamp looked like we were all set, but let us know if there's an outstanding question on this one - thanks!

Reply:
Nice one, I've decactivated on the staging, and installed the regular (non-pro) version and all looks good 👍